sdks/python/apache_beam/io/avroio_test.py

```diff
@@ -21,22 +21,27 @@
 import logging
 import math
 import os
-import sys
 import tempfile
 import unittest
 from builtins import range
+import sys
```

Comment:
I had a brief look at the test code -  there are a few `@unittest.skip("Skip during debugging")`, are you still working on this file, @fredo838?
Overall the test logic seems a little fragmented across 3 classes, and I think we could restructure the test for easier readability & maintenance. I'd have to take a closer look to have concrete suggestions for improvement. @RobbeSneyders if you have time, please take a look to reduce the review feedback loop since you are in the same timezone.